test/FastTests/Voron/EncryptionBufferPool.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Sparrow;
+using Sparrow.LowMemory;
+using Sparrow.Utils;
+using Voron.Impl;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.Voron
+{
+    public unsafe class EncryptionBufferPool : NoDisposalNeeded
+    {
+        public EncryptionBufferPool(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void dont_pool_buffers_larger_than_8Mb()
+        {
+            lock (this)
```

Comment:
Use a dedicated pool here, not the shared one

Reply:
done